techbuzz/src/main/java/com/sivalabs/techbuzz/posts/adapter/repositories/JooqPostRepository.java

```diff
@@ -121,6 +122,38 @@ public List<Post> findPosts(List<Long> postIds) {
                         r.value6()));
     }
 
+    @Override
+    public List<Post> findPostCreatedInNDays(int days) {
```

Comment:
Instead of days, we should pass `LocalDateTime` as input so that this method will be more flexible. You can get posts from any specific time instead of only by days.